libraries/AP_InertialSensor/AP_InertialSensor_MPU6000.cpp

```diff
@@ -595,7 +595,7 @@ bool AP_InertialSensor_MPU6000::_accumulate_fast_sampling(uint8_t *samples, uint
     }
 
     if (ret) {
-        float temp = (tsum/n_samples)/340.0 + 36.53;
```

Comment:
@rmackay9 I thought I heard you say we are somehow declaring these constants as floats instead of doubles in the compiler somehow?